inspector/src/components/actionTabs/tabs/propertyGrids/animations/addAnimation.tsx

```diff
@@ -312,15 +387,17 @@ export class AddAnimation extends React.Component<
         style={{ display: this.props.isOpen ? 'block' : 'none' }}
       >
         <div className='sub-content'>
-          <div className='label-input'>
-            <label>Target Path</label>
-            <input
-              type='text'
-              value={this.state.animationTargetPath}
-              onChange={(e) => this.handlePathChange(e)}
-              disabled
-            ></input>
-          </div>
+          {this.state.isUpdating ? null : (
+            <div className='label-input'>
+              <label>Target Path</label>
+              <input
+                type='text'
+                value={this.state.animationTargetPath}
+                onChange={(e) => this.handlePathChange(e)}

```

Comment:
is an arrow function needed here, or can that be `onChange={this.handlePathChange}`?

Can also be `onChange={this.handlePathChange.bind(this)}` if `this` context is needed (or make `handlePathChange` and arrow function?)

(I noticed it is done in different places in this code, I am just wondering whether or not the extra arrow function is needed)

Reply:
I'm actually using this across the editor to have access to the state or the event. Opted for the arrow function instead of the binding, mainly because in some other places I actually have to pass parameters...

Reply:
both bind and arrow functions to let you pass arguments, but this is totally fine as well. keep it